xs2a-adapter-aspsp-registry/src/main/java/de/adorsys/xs2a/adapter/registry/LuceneAspspRepository.java

```diff
@@ -222,4 +223,24 @@ private Query getNameQuery(String name) {
         }
         return find(queryBuilder.build(), after, size);
     }
+
+    @Override
+    public byte[] getAllRecords() {
+        List<Aspsp> storage = findAllMaxPageSize();
+
+        return storage.stream()
+            .map(this::toCsvString)
+            .collect(Collectors.joining("\n"))
+            .getBytes();
+    }
+
+
+    private String toCsvString(Aspsp aspsp) {
+        return aspsp.getId() + "," + checkStringForSpecialSymbol(aspsp.getName(), ",") + "," + aspsp.getBic() + "," + aspsp.getUrl() + ","
```

Comment:
The order of columns in the csv is contained in `AspspCsvRecord`. It's not a good idea to duplicate this knowledge in another place.